verify-cborld-presentation/src/pages/home.tsx

```diff
@@ -0,0 +1,27 @@
+import { Box, Link, StylesProvider } from "@material-ui/core";
+import React from "react";
+import styled from "styled-components";
+
+import { Description, DescriptionTitle, GenericButton } from "../components";
+
+export const HomePage: React.FC = () => {
```

Comment:
pages are also components, the same file naming applies to them too: HomePage.tsx